<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClustalWSupportRunDialog</class>
 <widget class="QDialog" name="ClustalWSupportRunDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>442</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>442</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Align with ClustalW</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="alignButton">
         <property name="text">
          <string>Align</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Advanced options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="gapOpenCheckBox">
            <property name="text">
             <string>Gap opening penalty</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="gapOpenSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>15.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="gapExtCheckBox">
            <property name="text">
             <string>Gap extension penalty</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="gapExtSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="value">
             <double>6.660000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QCheckBox" name="weightMatrixCheckBox">
            <property name="text">
             <string>Weight matrix</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="weightMatrixComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>IUB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CLUSTALW</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QCheckBox" name="maxIterationsCheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="text">
             <string>Max iterations</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="maxIterationsSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="iterationTypeComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>NONE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>TREE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ALIGNMENT</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="iterationTypeCheckBox">
            <property name="text">
             <string>Iteration type</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="outOrderCheckBox">
            <property name="text">
             <string>Out sequences order</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="outOrderComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Input</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Aligned</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="proteinGapParamGroupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Protein gap parameters</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="gapDistancesCheckBox">
            <property name="text">
             <string>Gap separation distance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="gapDistancesSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="hydrophilicGapsOffCheckBox">
          <property name="text">
           <string>Hydrophilic gaps off</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="endGapsCheckBox">
          <property name="text">
           <string>No end gap separation penalty</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="residueSpecificGapsOffCheckBox">
          <property name="text">
           <string>Residue-specific gaps off</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="inputGroupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="outputFileLabel">
            <property name="text">
             <string>Output file</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="outputFilePathButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="inputFileLabel">
            <property name="text">
             <string>Input file</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="outputFileLineEdit">
            <property name="maximumSize">
             <size>
              <width>16777214</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="inputFilePathButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="inputFileLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>gapOpenCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapOpenSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gapExtCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapExtSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gapDistancesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapDistancesSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxIterationsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxIterationsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weightMatrixCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>weightMatrixComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterationTypeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>iterationTypeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iterationTypeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxIterationsCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outOrderCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>outOrderComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
